test/e2e/daemon_set.go

```diff
@@ -61,8 +62,121 @@ var _ = Describe("Daemon set", func() {
 		f.afterEach()
 	})
 
-	It("should launch a daemon pod on every node of the cluster", func() {
-		testDaemonSets(f)
+	It("should run and stop simple daemon", func() {
+		ns := f.Namespace.Name
+		c := f.Client
+		simpleDSName := "simple-daemon-set"
+		image := "gcr.io/google_containers/serve_hostname:1.1"
+		label := map[string]string{daemonsetNameLabel: simpleDSName}
+
+		Logf("Creating simple daemon set %s", simpleDSName)
+		_, err := c.DaemonSets(ns).Create(&experimental.DaemonSet{
```

Comment:
nit: It would be nice to get rid of 'experimental' with a smarter import statement, so that if/when this gets promoted out of experimental, only the one import statement needs to change, not 10 lines of code.